eos-tech-support/eos-download-image

```diff
@@ -267,6 +288,10 @@ class EosDownloadImage(object):
                        help='Image version (default: newest)')
         g.add_argument('-I', '--iso', action='store_true',
                        help='Fetch an ISO image, not a raw disk image')
+        g.add_argument('-u', '--url',
```

Comment:
I think `--url` should be in the mutually-exclusive group `g` with `--mirror` and `--windows-tool`: this is conceptually a fourth mode, and this option really is mutually-exclusive with those two. (Ideally each mode would be a subparser but that ship sailed long ago!)